client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -63,21 +63,23 @@ const designSetup: Step = function DesignSetup( { navigation } ) {
 		} ),
 		[]
 	);
-	const isPrivateAtomic = Boolean(
-		site?.launch_status === 'unlaunched' && site?.options?.is_automated_transfer
-	);
 
+	const isAtomic = useSelect( ( select ) => site && select( SITE_STORE ).isSiteAtomic( site.ID ) );
+	const isPrivateAtomic = Boolean( site?.launch_status === 'unlaunched' && isAtomic );
+
+	const isEligibleForProPlan = useSelect(
+		( select ) => site && select( SITE_STORE ).isEligibleForProPlan( site.ID )
+	);
 	const showDesignPickerCategories =
 		isEnabled( 'signup/design-picker-categories' ) && ! isAnchorSite;
 	const showDesignPickerCategoriesAllFilter = isEnabled( 'signup/design-picker-categories' );
 	const showGeneratedDesigns =
 		isEnabled( 'signup/design-picker-generated-designs' ) && intent === 'build' && !! siteVertical;
 
 	const isPremiumThemeAvailable = Boolean(
-		useMemo(
-			() => sitePlanSlug && planHasFeature( sitePlanSlug, FEATURE_PREMIUM_THEMES ),
-			[ sitePlanSlug ]
-		)
+		useMemo( () => sitePlanSlug && planHasFeature( sitePlanSlug, FEATURE_PREMIUM_THEMES ), [
+			sitePlanSlug,
+		] )
```

Comment:
I think that CI may be complaining about this change?

Reply:
Yes but my prettier insists it's correct.

Reply:
Yes, mine too. Indeed, now it's complaining about types